5.application_scopes.md

```diff
@@ -66,185 +59,29 @@ The spec defines the constituent parts of a scope.
 
 | Attribute | Type | Required | Default Value | Description |
 |-----------|------|----------|---------------|-------------|
-| `type` | `string` | Y | | Determines type of the application scope. The core types are defined [above](#core-application-scope-types) |
 | `allowComponentOverlap` | `bool` | Y | | Determines whether a component is allowed to be in multiple instances of this scope type simultaneously. When false, the runtime implementation MUST produce an error and stop deployment if an attempt is made to place a component into more than one instance of this scope type simultaneously. |
-| `parameters` | [`[]Parameter`](#parameter) | N | | The scope's configuration options. |
+| `definitionRef` | [`DefinitionRef`](#definitionRef) | Y | | Index of the scope definition. |
 
-The `type` field defines the type name for this type. Examples may be found in the [Core Application Scope Types](#core-application-scope-types) above. Extended scope types MUST NOT be in the `core.oam.dev` namespace.  However, they must follow the [group, version, kind notation](2.overview_and_terminology.md). For example: `scopes.example.com/v1.ExtendedNetworkScope`
+### DefinitionRef
 
-#### Parameter
+We use the same [DefinitionRef](3.workload.md#DefinitionRef) in _WorkloadDefinition_.
 
-The parameters that a scope exposes to operators.
+## Core application scopes
```

Comment:
I suggest we don't define core application scopes for now? Either Network Scope or Health Scope does not sound like a MUST implement to me. Maybe move them to standard scope @ryanzhang-oss

Reply:
totally agree

Reply:
agree